certora/specs/ERC20FlashMint.spec

```diff
@@ -0,0 +1,43 @@
+import "helpers.spec"
+import "methods/IERC20.spec"
+import "methods/IERC3156.spec"
+
+methods {
+    // non standard ERC3156 functions
+    flashFeeReceiver() returns (address) envfree
+
+    // patched - function summaries below
+    _mint(address account, uint256 amount) => specMint(account, amount)
+    _burn(address account, uint256 amount) => specBurn(account, amount)
```

Comment:
Does this also replace the behavior when the function is called internally?

Reply:
Yes.